Sources/ActomatonUI/Store/Internal/StoreCore.swift

```diff
@@ -19,18 +18,17 @@ internal final class StoreCore<Action, State, Environment>
     internal init(
         state initialState: State,
         reducer: Reducer<Action, State, Environment>,
-        environment: Environment
+        environment: Environment,
+        configuration: StoreConfiguration
     )
     {
         self._state = CurrentValueSubject(initialState)
-        self.reducer = reducer
         self.environment = environment
 
         self.actomaton = MainActomaton(
             state: initialState,
-            reducer: lift(reducer: Reducer { action, state, environment in
-                reducer.run(action, &state, environment)
-            }),
+            reducer: lift(reducer: reducer)
+                .log(format: configuration.logFormat),
```

Comment:
NOTE:
While developer can customize `reducer` itself, internally-`lift`-ed reducer can't be customized, thus introducing `configuration.logFormat` is required.